package.json

```diff
@@ -47,6 +47,7 @@
     "express": "^4.16.4",
     "please-upgrade-node": "^3.2.0",
     "raw-body": "^2.3.3",
+    "serverless-http": "^2.3.0",
```

Comment:
shouldn't this be a dev dependency? it's only needed for testing

Reply:
Apparently I forgot `npm` commands.